doc/bc/5.90/relation_indexing.md

```diff
@@ -0,0 +1,13 @@
+# Relation search indexing changes 
+
+For the 2018.06 release, eZ Publish changed the way relations are indexed. Previously the relation metadata was loaded recursively, meaning that relations of relations etc. were also included. This behavior was changed to only include Objects one level deep so only direct relations are taken into account.
```

Comment:
should explain why the change was done.

like `are taken into account, in order to solve various recursion problems when relations are circular.`

Reply:
I added another sentence for the "why" part.